alembic/versions/253e7b76eb8e_modify_assignee.py

```diff
@@ -0,0 +1,40 @@
+"""modify assignee
+
+Revision ID: 253e7b76eb8e
+Revises: 1ea1e8070ac8
+Create Date: 2016-04-11 20:15:52.864916
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '253e7b76eb8e'
+down_revision = '1ea1e8070ac8'
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+import sqlalchemy_utils
+
+
+def upgrade():
+    op.add_column('assignee', sa.Column('current', sa.Boolean(), nullable=True))
```

Comment:
Add a check constraint to ensure the value is never `false`. That leaves `null` and `true` as the permissible values.